### PR TITLE
create first-steps.md

### PR DESCRIPTION
**Pull request by ArranHL**
@dogi this is the more streamlined version that will need additional supporting documents, let me know if this is still too much content to include and I'll strip down further

### CODE REVIEW

**Comment by ArranHL** [comment]
First supporting image is Files for New10Steps #14

**Comment by dogi** [comment]
found the file ... it is in the root ... we should change that ...

anyhow https://raw.githack.com/treehouses/treehouses.github.io/slimNew10Steps/#!rpi-vi-first-steps.md

**Comment by ArranHL** [comment]
hi @dogi I was going to create a "vi" folder in pages(folder) but thought that you could not create an empty folder without first having a file to add to it, should of created #11 directly in pages/vi as a new file

Should I do a new PR of is there a way of changing the destination?

**Comment by ArranHL** [comment]
@dogi nvm got it :)

**Comment by dogi** [comment]
generally like what I see
still have not gone totally over it

**Comment by ArranHL** [comment]
@dogi awesome, I've put together a very brief little flow chart that you can access here: https://drive.google.com/file/d/1ODMvX6IY_QQs_0fdRc4vSMO1dqL3hfA0/view?usp=sharing 

If you'd like to add any specific ideas.
image

I was thinking of an introductory step specifically focused on getting them more involved with Cli

**Comment by dogi** [comment]
https://raw.githack.com/treehouses/treehouses.github.io/slimNew10Steps/#!pages/vi/rpi-vi-first-steps.md

File: pages/vi/rpi-vi-first-steps.md [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+
+If you are selected for the internship after completing the steps, you will be officially invited to join the OLE interns team! We’ll add you to the Virtual Interns Gitter chatroom and assign you to a specific team to work on developing and improving OLE’s software.
+
+Once accepted, you and your team will work on an assignment, and we’ll switch up the assignments each week. As part of this internship, you will have the opportunity to work with software and languages including **[Git](https://git-scm.com/)**, **[GitHub](https://github.com/)**, **[Markdown](https://daringfireball.net/projects/markdown/syntax)**, **[Vagrant](https://www.vagrantup.com/)**, **[VirtualBox](https://www.virtualbox.org/)**, **[Command Line/Terminal](https://en.wikipedia.org/wiki/Command-line_interface)**, **[Command Line/Terminal Scripts](https://www.codecademy.com/articles/command-line-commands)**, **[Vim](https://www.vim.org/)**, **[CouchDB](http://couchdb.apache.org/)** and **[Docker](https://www.docker.com/)**.
```
Comment (dogi) [inline pages/vi/rpi-vi-first-steps.md]: and hardware https://raspberrypi.org

File: pages/vi/rpi-vi-first-steps.md [hunk 2]
```diff
@@ -0,0 +1,238 @@
+# First Steps
+
+### The Big Picture
+
+Welcome to the first steps for becoming an OLE Virtual Intern! We treat these first steps as a vetting process to prove that you can follow simple instructions before you can progress to working on harder projects in bigger teams. Consider them to be the interview for the internship.
```
Comment (dogi) [inline pages/vi/rpi-vi-first-steps.md]: OLE Virtual Intern => Virtual Systems Engineering Intern @ OLE project treehouses.io

File: pages/vi/rpi-vi-first-steps.md [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+The goal of OLE is to provide educational material to learners and educators in remote areas of the globe and the Raspberry Pi is utilized in pursuit of this goal to provide a means by which devices can access educational resources.
+Our goal is to provide a stable system that is widely accessible by numerous devices and operating systems.
+This equipment will be deployed in the field with little or no access to the internet, so its reliability is of paramount importance.
+
+
+**NOTE**: This is an unpaid but intensive internship that requires 16 hours of work each week. More information about the internship can be found in our [FAQ](vi-faq.md#General_Internship_Questions). If you have further questions, don’t hesitate to ask other members in the Gitter [chat](vi-chat.md)!
+
+### Pre-Requisites
+#### Hardware requirement:
+1.  A Raspberry Pi 4
```
Comment (dogi) [inline pages/vi/rpi-vi-first-steps.md]: Raspberry Pi 4b  link to https://www.raspberrypi.org/products/raspberry-pi-4-model-b/ (all rpis 3 and greater work)

File: pages/vi/rpi-vi-first-steps.md [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+1.  A power cord for the Pi
+	- Standard micro USB power cord: 15.3W for Pi 4
+1.  3 Class 10 microSD cards
+	- minimal 16GB, but we recommend 32GB or greater
```
Comment (dogi) [inline pages/vi/rpi-vi-first-steps.md]: 16gb (and greater) but we recommend U1 speed class

File: pages/vi/rpi-vi-first-steps.md [hunk 5]
```diff
@@ -0,0 +1,238 @@
...
+1.  A microSD card reader
+	- Check your computer if you have a slot for a card reader otherwise use a USB reader
+1.  Ethernet cable or wifi dongle
```
Comment (dogi) [inline pages/vi/rpi-vi-first-steps.md]: wifi dongle? plz remove

File: pages/vi/rpi-vi-first-steps.md [hunk 6]
```diff
@@ -0,0 +1,238 @@
...
+1.  USB keyboard, monitor, and HDMI cable
```
Comment (dogi) [inline pages/vi/rpi-vi-first-steps.md]: please move this to last place and with optional

File: pages/vi/rpi-vi-first-steps.md [hunk 7]
```diff
@@ -0,0 +1,238 @@
...
+	- Optional if using your own laptop via VNC
+1.  8 GB of hard-drive space
```
Comment (dogi) [inline pages/vi/rpi-vi-first-steps.md]: remove ;)

File: pages/vi/rpi-vi-first-steps.md [hunk 8]
```diff
@@ -0,0 +1,238 @@
...
+1.  8 GB of memory (RAM)
```
Comment (dogi) [inline pages/vi/rpi-vi-first-steps.md]: yes like this
but we should write we suggest a laptop with 8 and more gb ram

File: pages/vi/rpi-vi-first-steps.md [hunk 9]
```diff
@@ -0,0 +1,238 @@
...
+
+There is no official deadline to complete these steps, but most candidates who are approved for the internship program finished the steps within 7-8 days. Good luck!
+
+-------------------------------------------------------------------------------
+
+## Step 0 - Introductory Steps
+
+1. Complete the [New Virtual Intern Application form](https://docs.google.com/forms/d/e/1FAIpQLSeooEi9vJeb_nsf8WEXqEVRn1Ko6d4M-qF76ik4v0xgToUiSA/viewform?usp=sf_link).
+
+2. Create a [GitHub account](https://github.com/) then watch <a href="https://github.com/open-learning-exchange/open-learning-exchange.github.io" target="_blank">Open Learning Exchange's github.io repo</a> and Follow [Leonard](https://github.com/leomaxi), [Dogi](https://github.com/dogi), [Laxman](https://github.com/lmmrssa), [Paul](https://github.com/paulbert), [Mappuji](https://github.com/empeje), [Ignacio](https://github.com/i5o), [Arpita](https://github.com/singharpita), [Arran](https://github.com/ArranHL), [Sam](https://github.com/Liew211) and [Roshan](https://github.com/rrijal53) on GitHub.
```
Comment (dogi) [inline pages/vi/rpi-vi-first-steps.md]: we should add mostly active treehouses people

Comment (dogi) [inline pages/vi/rpi-vi-first-steps.md]: treehouses github please

File: pages/vi/rpi-vi-first-steps.md [hunk 10]
```diff
@@ -0,0 +1,238 @@
...
+
+
+3. We use Crowdin to translate our user interface into many different languages. If you know another language besides English, we would greatly appreciate if you would help translate. To do so, go to [this Crowdin link](https://crowdin.com/project/ole-planet/invite) to create a Crowdin account. A good goal is to get 10% of the words in your chosen language translated. If you have time, it would be even better to translate more of the words. After finishing [Step 4 - Planet Tutorial](vi-first-steps.md#Step_4_-_Planet_Tutorial), check out this [link](https://crowdin.com/page/tour#tab_translators) to learn how you can start contributing on Crowdin.   
+
+**NOTE**: See the [Planet User Manual](#!./pages/techgenius/tg-planet-user-manual.md) for an overview of functionality and an explanation of various terms. Your contributions will not only help us out but also give you a better understanding of what Planet does.
```
Comment (dogi) [inline pages/vi/rpi-vi-first-steps.md]: remove

File: pages/vi/rpi-vi-first-steps.md [hunk 11]
```diff
@@ -0,0 +1,238 @@
...
+
+In GitHub, we "[Watch](https://help.github.com/en/articles/watching-and-unwatching-repositories)" organization repositories and ["Follow"](https://help.github.com/en/articles/following-people) individuals. Look for the "Watch" button at the top right of the page when you are in a repository.  Also, follow each other to see what others in the group are doing. (Click on the image to enlarge.)
+
+
+3. We use Crowdin to translate our user interface into many different languages. If you know another language besides English, we would greatly appreciate if you would help translate. To do so, go to [this Crowdin link](https://crowdin.com/project/ole-planet/invite) to create a Crowdin account. A good goal is to get 10% of the words in your chosen language translated. If you have time, it would be even better to translate more of the words. After finishing [Step 4 - Planet Tutorial](vi-first-steps.md#Step_4_-_Planet_Tutorial), check out this [link](https://crowdin.com/page/tour#tab_translators) to learn how you can start contributing on Crowdin.
```
Comment (dogi) [inline pages/vi/rpi-vi-first-steps.md]: remove?

**Review by dogi — CHANGES_REQUESTED** [review]
@ArranHL good start ;) still too much open-learning-exchange :P

File: pages/vi/rpi-vi-first-steps.md [hunk 12]
```diff
@@ -0,0 +1,236 @@
...
+# Step 2 - Docker
+*NEW DOCKER .md PROCESS NEEDED* 
+ **Docker** is a computer program that performs operating-system-level virtualization also known as containerization. In this section, you will learn the basics of interacting with Docker and Docker Compose through the command-line interface and basic commands for maintaining your Planet installation.
+
+-----------------------------------------------------------------------
+
+# Step 3 - Markdown and Fork Tutorial
+*NEW Fork PROCESS NEEDED* 
+Follow the instructions on the NEW .md PROCESS NEEDED  
+Make sure that you've linked to your github.io and pull request link on the [Gitter chat](https://gitter.im/open-learning-exchange/chat) (https://YourUserName.github.io and `LinkToYourPullRequest`). Also include Raw.githack link in your pull request(https://raw.githack.com/YourUserName/YourUserName.github.io/YourBranchName/#!pages/vi/profiles/YourUserName.md).
```
Comment (dogi) [inline pages/vi/rpi-vi-first-steps.md]: treehouses Lobby

File: pages/vi/rpi-vi-first-steps.md [hunk 13]
```diff
@@ -0,0 +1,236 @@
...
+
+# Step 5 - Keeping Fork Updated
+
+Follow the directions at NEW .md PROCESS NEEDED to keep your username.github.io and your local repository up to date.
+
+-----------------------------------------------------------------------
+
+# Step 6 - GitHub Issues Tutorial
+
+Follow the tutorial under the NEW .md PROCESS NEEDED to create at least one issue. Post a link in [Gitter](https://gitter.im/open-learning-exchange/chat) whenever you create an issue or when you comment on someone else's issue. You are encouraged to post as many issues as you can for improving the page as well as for personal practice. No issue is too big or too small to be filed and it is OK if you are not sure how to fix it yourself. If you know how to solve an issue, be sure to provide a detailed account of your research and show how to fix it. It is ok to file an issue about minor typos and very small changes, but do not make this the case for all of the issues that you file. You can also work on issues that you didn't create.
```
Comment (dogi) [inline pages/vi/rpi-vi-first-steps.md]: treehouses Lobby

File: pages/vi/rpi-vi-first-steps.md [hunk 14]
```diff
@@ -0,0 +1,236 @@
...
+
+The MDwiki has plenty of resources to help you complete the steps. There is a list of useful links at the end of each step. We also created a [FAQ page](https://open-learning-exchange.github.io/#!./pages/vi/vi-faq.md) where you can find the answers to some commonly asked questions. This page has even more [useful links and video tutorials](https://open-learning-exchange.github.io/#!./pages/vi/vi-faq.md#Helpful_Links) that will help you become familiar with the tools/languages we use. For anything that is not on the FAQ page, Google and Stack Exchange are your friends :)
```
Comment (dogi) [inline pages/vi/rpi-vi-first-steps.md]: no ole.gh.io

File: pages/vi/rpi-vi-first-steps.md [hunk 15]
```diff
@@ -0,0 +1,236 @@
...
+
+**We also want you to keep us updated a relatively good amount in the Gitter chat as you complete these steps. We will ask you to send us messages, links, and screenshots along the First Steps, and we will use this to track your progress, so please do not forget to do so.** Check our [Gitter chat](https://open-learning-exchange.github.io/#!./pages/vi/vi-chat.md) page for more information about how to best communicate in Gitter.
```
Comment (dogi) [inline pages/vi/rpi-vi-first-steps.md]: no ole.gh.io

**Review by dogi — APPROVED** [review]

File: pages/vi/rpi-vi-first-steps.md [hunk 16]
```diff
@@ -0,0 +1,215 @@
...
+The goal of OLE is to provide educational infrastructure to learners and educators in remote areas of the globe. The Raspberry Pi is utilized in pursuit of this goal to provide a means by which devices can access educational resources.
+Our goal is to provide a stable system that is widely accessible by numerous devices and operating systems.
+This equipment will be deployed in the field with little or no access to the internet, so its reliability is of paramount importance.
+
+
+**NOTE**: This is an unpaid but intensive internship that requires 16 hours of work each week. More information about the internship can be found in our [FAQ](vi-faq.md#General_Internship_Questions). If you have further questions, don’t hesitate to ask other members in the Gitter [chat](vi-chat.md)!
+
+### Pre-Requisites
+#### Hardware requirement:
+1.  A Raspberry Pi 4 ([Pi 4 information](https://www.raspberrypi.org/products/raspberry-pi-4-model-b/))
```
Comment (dogi) [inline pages/vi/rpi-vi-first-steps.md]: rpi3 s are missing

File: pages/vi/rpi-vi-first-steps.md [hunk 17]
```diff
@@ -0,0 +1,215 @@
...
+2.  A power cord for the Pi
+	- Standard micro USB power cord: 15.3W for Pi 4
```
Comment (dogi) [inline pages/vi/rpi-vi-first-steps.md]: usb-c for rpi4b

File: pages/vi/rpi-vi-first-steps.md [hunk 18]
```diff
@@ -0,0 +1,215 @@
...
+3.  3 Class 10 microSD cards
+	- 16gb or greater
+	- Recommend U1 speed class
+4.  A microSD card reader
+	- Check your computer if you have a slot for a card reader otherwise use a USB reader
+5.  Ethernet cable
+6.  USB keyboard, monitor, and HDMI cable
```
Comment (dogi) [inline pages/vi/rpi-vi-first-steps.md]: plz but under optional on the end of this list

File: pages/vi/rpi-vi-first-steps.md [hunk 19]
```diff
@@ -0,0 +1,215 @@
...
+
+# Step 2 - Docker
+*NEW DOCKER .md PROCESS NEEDED* 
+ **Docker** is a computer program that performs operating-system-level virtualization also known as containerization. In this section, you will learn the basics of interacting with Docker and Docker Compose through the command-line interface and basic commands for maintaining your Planet installation.
+
+
+# Step 3 - Markdown and Fork Tutorial
+*NEW Fork PROCESS NEEDED* 
+Follow the instructions on the NEW .md PROCESS NEEDED  
+Make sure that you've linked to your github.io and pull request link on the [Gitter chat](https://gitter.im/open-learning-exchange/treehouses) (https://YourUserName.github.io and `LinkToYourPullRequest`). Also include Raw.githack link in your pull request(https://raw.githack.com/YourUserName/YourUserName.github.io/YourBranchName/#!pages/vi/profiles/YourUserName.md).
```
Comment (dogi) [inline pages/vi/rpi-vi-first-steps.md]: gitter Lobby

and

link needs to have .../Lobby

**Review by dogi — APPROVED** [review]
@samuelchen1213 we are close ;)

**Review by dogi — APPROVED** [review]

**Review by samuelchen1213 — APPROVED** [review]

**Pull request merged**